### PR TITLE
Fix rimraf on windows for prebuild-sample

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
https://github.com/isaacs/rimraf/blob/master/README.md

"CLI

If installed with npm install rimraf -g it can be used as a global command rimraf <path [<path ...] which is useful for cross platform support."

### CODE REVIEW

**Comment by CodeBaboon** [comment]
LGTM

**Comment by dlockhart** [comment]
This is still failing for me on Windows. It's actually the `mkdir ./sample/dist` that's failing with `The syntax of the command is incorrect.`, because it's expecting slashes that go the opposite way. I'd recommend having a look at [mkdir-p](https://www.npmjs.com/package/mkdir-p) as an alternative to the native `mkdir`.

**Comment by jstefaniuk-d2l** [comment]
Ohh, are you using cmd? I was using the git bash and never got that error, I'll look into that package

**Comment by jstefaniuk-d2l** [comment]
Tested on git bash, the mac terminal, and windows cmd

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "React component for viewing files",
   "main": "src/fileViewer.js",
   "scripts": {
-    "prebuild-sample": "rimraf ./sample/dist && mkdir ./sample/dist",
+    "prebuild-sample": "rimraf ./sample/dist .\\sample\\dist && mkdirp ./sample/dist",
```
Comment (dlockhart) [inline package.json]: I don't think you need the second argument to rimraf (it worked fine for me on Windows). This should work:

    rimraf ./sample/dist && mkdirp ./sample/dist

**Pull request merged**